lib/models/Orders.js

```diff
@@ -1,7 +1,19 @@
 module.exports = (sequelize, DataTypes) => {
   const Orders = sequelize.define('Orders', {
-    data: {
+    name: {
       type: DataTypes.STRING
+    },
+    location: {
+      type: DataTypes.STRING
+    },
+    phone: {
+      type: DataTypes.INTEGER
+    },
+    email: {
+      type: DataTypes.STRING
+    },
+    cost: {
```

Comment:
I wonder if this should be *total_cost* for clarity, given that we're going to have *cost* on lots of different tables. It would be clearer that this is the sum of the cost of all items associated with this order, but at the same time having just one variable to remember is good. Just bringing it up.

Reply:
Maybe just `total` for this field?  `total_cost` might be good.